db/db_compaction_test.cc

```diff
@@ -155,15 +155,17 @@ class DBCompactionDirectIOTest : public DBCompactionTest,
 
 class DBCompactionWaitForCompactTest
     : public DBTestBase,
```

Comment:
Would you mind adding a comment for the test to describe what each bool parameter is?

Reply:
The three params match exactly what `WaitForCompactOptions` has and details are in the options.h file, so I found that it may not be necessary, but I can add a quick comment to say see `WaitForCompactOptions` for details about the three.